# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

MoveDogAwayActionGoal action_goal
MoveDogAwayActionResult action_result
MoveDogAwayActionFeedback action_feedback
